shared/data/tracker_lists/trackersWithParentCompany.json

```diff
@@ -7190,269 +7190,74 @@
         }
     },
     "Social": {
-        "addthis.com": {
```

Comment:
I'm removing all the social entries that we're not currently using through the easylist. We can add these back later if we need to.